bowtie.wdl

```diff
@@ -0,0 +1,43 @@
+task Bowtie_se {
+    File reference
+    File read1
+    String sam
+    String? precommand
+    Int? threads
+    String? otherOptions
+
+    command {
+        ## https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/
+        set -eo pipefail
+        mkdir -p
+        ${precommand}
+        bowtie \
+        ${otherOptions}
+        ${"--threads" + threads} \
+        ${reference} \
+        ${read1} > \
+        ${sam}
+    }
+}
+
+task Bowtie_pe {
```

Comment:
Isn't it possible to make a generic bowtie task for both paired and single end?

Reply:
I could give it a try